internal/factory/container/container_test.go

```diff
@@ -582,7 +582,7 @@ var _ = t.Describe("Container", func() {
 			serverCaps := []string{}
 
 			Expect(sut.SpecSetupCapabilities(caps, serverCaps, false)).To(BeNil())
-			verifyCapValues(sut.Spec().Config.Process.Capabilities, len(capability.List())-1)
+			verifyCapValues(sut.Spec().Config.Process.Capabilities, int(validate.LastCap())+1-1)
```

Comment:
I don't get the "+1-1" statement. Isn't it a no-op ?

Reply:
> I don't get the "+1-1" statement. Isn't it a no-op ?

From the final result, yes, it can be simply written as `int(validate.LastCap())`. 
This is mainly to avoid confusion because `int(validate.LastCap())+1` represents the number of `ALL` capabilities in the current environment, and `-1` indicates removing `CHOWN` from `ALL`.
Without the `-1`, the idea of excluding `CHOWN` from `ALL` in the test case cannot be reflected.

Reply:
Got it. I think a comment would be clearer then, but that's a nit as the Add/Remove capabilities is just a couple lines above.
Thank you!

Reply:
yeah  I agree would you mind adding a comment justifying it @whygyc ?

Reply:
Sure, I agree. I will add a comment as you suggested.